english/turing-test/turing-test.md

```diff
@@ -24,4 +24,8 @@ Do you:
 
 [Forget ELIZA, ask Siri](siri/ask_siri.md)
 
-[Wear the one ring and turn invisible](ring/one_ring.md)
\ No newline at end of file
+[Wear the one ring and turn invisible](ring/one_ring.md)
+
+[Forget Siri, asks Cortana](Cortana/ask_Cortana.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.